sentry-android-core/src/main/java/io/sentry/android/core/ManifestMetadataReader.java

```diff
@@ -227,6 +230,17 @@ static void applyMetadata(
 
         options.setProguardUuid(
             readString(metadata, logger, PROGUARD_UUID, options.getProguardUuid()));
+
+        final SdkVersion sdkInfo = options.getSdkVersion();
+        if (sdkInfo != null) {
+          sdkInfo.setName(readStringNotNull(metadata, logger, SDK_NAME, sdkInfo.getName()));
+          sdkInfo.setVersion(
+              readStringNotNull(metadata, logger, SDK_VERSION, sdkInfo.getVersion()));
+          options.setSdkVersion(sdkInfo);
+        } else {
+          // Should always be set by the Options constructor but let's error out if that changes.
+          throw new IllegalArgumentException("Property options.sdkVersion must not be null.");
```

Comment:
This would crash the SDK and we don't want to do that, degrade gracefully.
I'd rather create a new `SdkVersion` instance and set it.

Reply:
OK, changed